test/serena/test_tool_parameter_types.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from serena.mcp import SerenaMCPFactorySingleProcess
+from serena.config.serena_config import SerenaConfig
+
+def load_agent():
```

Comment:
should rather be load_config, but I think we don't need this function at all. You only have one test, pls create the config there. You should also not load it from disc but instantiate it directly with gui and dashboard disabled, like

```
SerenaConfig(gui_log_window_enabled=False, web_dashboard=False, log_level=logging.ERROR)
```